acceptance_tests/__init__.py

```diff
@@ -9,10 +10,16 @@
 logging.basicConfig()
 configure(logger_factory=LoggerFactory())
 
-if os.getenv('HEADLESS', 'True') == 'True':
-    chrome_options = webdriver.ChromeOptions()
-    chrome_options.add_argument("--no-sandbox")
-    chrome_options.add_argument("--window-size=1920,1200")
-    browser = Browser('chrome', headless=True, options=chrome_options)
-else:
-    browser = Browser('chrome')
+
+@retry(retry_on_exception=lambda e: isinstance(e, ConnectionError), wait_fixed=1000, stop_max_attempt_number=30)
+def create_browser():
+    if os.getenv('HEADLESS', 'True') == 'True':
+        chrome_options = webdriver.ChromeOptions()
+        chrome_options.add_argument("--no-sandbox")
+        chrome_options.add_argument("--window-size=1920,1200")
```

Comment:
If chrome is running in headless mode, why are we setting the size of the window?

Reply:
The app isn't yet responsive so there were tests failing. It might be ok now